modules/core/src/core/animation-loop.js

```diff
@@ -228,6 +228,12 @@ export default class AnimationLoop {
     return this.props.onInitialize(...args);
   }
 
+  // Called on each frame, can be overridden to call onRender multiple times
+  // to support e.g. stereoscopic rendering
+  onFrame(...args) {
```

Comment:
`onRender` vs `onFrame` is confusing.

Reply:
> onRender vs onFrame is confusing.

OK. We can address that in the followup PR that adds display support to the main animation loop.